### PR TITLE
Add fuwwy twanswation

### PR DESCRIPTION
**Pull request by ghostoblivion**
OwO

### CODE REVIEW

**Comment by TheNoob27** [comment]
sigh, guess it's time to mewge this

**Comment by TheNoob27** [comment]
before i merge this, go to the other files and add FUWWY: "Fuwwy" to LANGUAGES

File: languages/en-GB.js [hunk 1]
```diff
@@ -725,5 +725,6 @@ module.exports = {
     EN_US: "English (US)",
     LT_LT: "Lithuanian",
     TR_TR: "Turkish",
+	FUWWY: "Fuwwy",
```
Comment (TheNoob27) [inline languages/en-GB.js:728]: fix indent thankies

**Review by CircuitSacul — APPROVED** [review]

**Comment by TheNoob27** [comment]
turns out im stupid and you actually shouldn't really have added fuwwy to all the other languages except english, since that field is for languages in the files language "eg Turkish in Turkish", but it'll do for now

**Comment by ghostoblivion** [comment]
> turns out im stupid and you actually shouldn't really have added fuwwy to all the other languages except english, since that field is for languages in the files language "eg Turkish in Turkish", but it'll do for now

lmaoooooo

**Comment by TheNoob27** [comment]
but you can - if you want - put the languages in your fuwwy language (Tuwkish or whatever)

**Pull request merged**